src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.html

```diff
@@ -184,6 +184,12 @@
                               [selectionLimit]="4"
                               (selection)="appSelection()">
             </cd-select-badges>
+            <i *ngIf="getApplicationWarning()"
```

Comment:
Provided thta function calls in templates [should be avoided when possible](https://medium.com/showpad-engineering/why-you-should-never-use-function-calls-in-angular-template-expressions-e1a50f9c0496),
why not just:
```suggestion
            <i *ngIf="data.applications.selected <= 0"
```

Reply:
Yeah agree, should have avoided them